Jenkinsfile

```diff
@@ -112,7 +112,7 @@ for (String os in runITsOses) {
                                 }
                             }
                         } finally {
-                            //junit testResults: '**/core-it-suite/**/target/surefire-reports/*.xml', allowEmptyResults: true                                            
+                            junit testResults: '**/core-it-suite/**/target/surefire-reports/*.xml', allowEmptyResults: true
```

Comment:
What with standard path, like `**target/surefire-reports/*.xml` and for failsafe?

Reply:
I am not sure. I assumed these patterns were tested before and it the junit step was accidentally left commented out.

I think we could go with `**target/surefire-reports/*.xml`. It will pick-up some test results from `core-it-support`, but that may be desirable.

There are not `failsafe-reports` directories, but to be safe we could include those as well.